rosidl_generator_rs/cmake/rosidl_generator_rs_generate_interfaces.cmake

```diff
@@ -119,6 +119,7 @@ foreach(dep ${target_dependencies})
 endforeach()
 
 set(generator_arguments_file "${CMAKE_CURRENT_BINARY_DIR}/rosidl_generator_rs__arguments.json")
+set(package_version "${${PROJECT_NAME}_VERSION}")
```

Comment:
Instead of using a separate variable for storing the package version, you can reuse the arguments file and write the value of `${${PROJECT_NAME}_VERSION}` there.

Reply:
I appended package version into generator argument file in https://github.com/ros2-rust/ros2_rust/pull/191/commits/0c78e272cb15d3996ef2bb0246e4ac1267cee145.

Reply:
sample json
```
{
  "package_name": "action_msgs",
  "output_dir": "/home/daisuke/workspace/ros2_rust_ws/build/action_msgs/rosidl_generator_rs/action_msgs",
  "template_dir": "/home/daisuke/workspace/ros2_rust_ws/install/rosidl_generator_rs/share/rosidl_generator_rs/cmake/../resource",
  "idl_tuples": [
    "/home/daisuke/workspace/ros2_rust_ws/build/action_msgs/rosidl_adapter/action_msgs:msg/GoalInfo.idl",
    "/home/daisuke/workspace/ros2_rust_ws/build/action_msgs/rosidl_adapter/action_msgs:msg/GoalStatus.idl",
    "/home/daisuke/workspace/ros2_rust_ws/build/action_msgs/rosidl_adapter/action_msgs:msg/GoalStatusArray.idl",
    "/home/daisuke/workspace/ros2_rust_ws/build/action_msgs/rosidl_adapter/action_msgs:srv/CancelGoal.idl"
  ],
  "ros_interface_files": [
    "/home/daisuke/workspace/ros2_rust_ws/build/action_msgs/rosidl_adapter/action_msgs/msg/GoalInfo.idl",
    "/home/daisuke/workspace/ros2_rust_ws/build/action_msgs/rosidl_adapter/action_msgs/msg/GoalStatus.idl",
    "/home/daisuke/workspace/ros2_rust_ws/build/action_msgs/rosidl_adapter/action_msgs/msg/GoalStatusArray.idl",
    "/home/daisuke/workspace/ros2_rust_ws/build/action_msgs/rosidl_adapter/action_msgs/srv/CancelGoal.idl"
  ],
  "ros_interface_dependencies": [
    "builtin_interfaces:/home/daisuke/workspace/ros2_rust_ws/install/builtin_interfaces/share/builtin_interfaces/msg/Duration.idl",
    "builtin_interfaces:/home/daisuke/workspace/ros2_rust_ws/install/builtin_interfaces/share/builtin_interfaces/msg/Time.idl",
    "unique_identifier_msgs:/opt/ros/humble/share/unique_identifier_msgs/msg/UUID.idl"
  ],
  "target_dependencies": [
    "/home/daisuke/workspace/ros2_rust_ws/install/rosidl_generator_rs/share/rosidl_generator_rs/cmake/../../../lib/rosidl_generator_rs/rosidl_generator_rs",
    "/home/daisuke/workspace/ros2_rust_ws/install/rosidl_generator_rs/share/rosidl_generator_rs/cmake/../../../local/lib/python3.10/dist-packages/rosidl_generator_rs/__init__.py",
    "/home/daisuke/workspace/ros2_rust_ws/install/rosidl_generator_rs/share/rosidl_generator_rs/cmake/../resource/msg.rs.em",
    "/home/daisuke/workspace/ros2_rust_ws/install/rosidl_generator_rs/share/rosidl_generator_rs/cmake/../resource/srv.rs.em",
    "/home/daisuke/workspace/ros2_rust_ws/build/action_msgs/rosidl_adapter/action_msgs/msg/GoalInfo.idl",
    "/home/daisuke/workspace/ros2_rust_ws/build/action_msgs/rosidl_adapter/action_msgs/msg/GoalStatus.idl",
    "/home/daisuke/workspace/ros2_rust_ws/build/action_msgs/rosidl_adapter/action_msgs/msg/GoalStatusArray.idl",
    "/home/daisuke/workspace/ros2_rust_ws/build/action_msgs/rosidl_adapter/action_msgs/srv/CancelGoal.idl",
    "/home/daisuke/workspace/ros2_rust_ws/build/action_msgs/rosidl_adapter/action_msgs/msg/GoalInfo.idl",
    "/home/daisuke/workspace/ros2_rust_ws/build/action_msgs/rosidl_adapter/action_msgs/msg/GoalStatus.idl",
    "/home/daisuke/workspace/ros2_rust_ws/build/action_msgs/rosidl_adapter/action_msgs/msg/GoalStatusArray.idl",
    "/home/daisuke/workspace/ros2_rust_ws/build/action_msgs/rosidl_adapter/action_msgs/srv/CancelGoal.idl",
    "/home/daisuke/workspace/ros2_rust_ws/install/builtin_interfaces/share/builtin_interfaces/msg/Duration.idl",
    "/home/daisuke/workspace/ros2_rust_ws/install/builtin_interfaces/share/builtin_interfaces/msg/Time.idl",
    "/opt/ros/humble/share/unique_identifier_msgs/msg/UUID.idl"
  ],
  "package_version": "1.2.0"
}
```